pkgs/data/icons/buuf-icon-theme/default.nix

```
@@ -0,0 +1,28 @@
+{ lib, stdenvNoCC, fetchzip }:
+
+stdenvNoCC.mkDerivation {
+  pname = "buuf-icon-theme";
+  version = "3.45";
+
+  src = fetchzip {
+    url = "http://buuficontheme.free.fr/buuf3.45.tar.xz";
+    sha256 = "sha256-TrUHW3W4fMR8esirEenKxV+cxnzjC5U0raPfutear+g=";
```

`hash` and SRI hashes are the way these are used these days.

```suggestion
    hash = "sha256-TrUHW3W4fMR8esirEenKxV+cxnzjC5U0raPfutear+g=";
```